### PR TITLE
[INFRA-21463] netbeans.org transition explanation

### PR DESCRIPTION
**Pull request by vieiro**

### CODE REVIEW

**Review by JaroslavTulach — APPROVED** [review]
Great summary of the backups available at one place. Thank you!

**Comment by jtulach** [comment]
Should this page be linked from some other to form a "web"?

File: netbeans.apache.org/src/content/about/oracle-transition.asciidoc [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+:description: Apache NetBeans History
+:toc: left
+:toc-title:
+:icons: font
+
+== Standing on our own feet
+
+In 2016 Oracle donated the source code of NetBeans to the Apache Software Foundation (see xref:history.asciidoc[Our History] for details).
+
+During the next five years Oracle has also been supporting the NetBeans Project with some server infraestructure (and a great team!). This has been of course of great help, and has allowed us to move forward. We want to thank Oracle for all this help.
```
Comment (KacerCZ) [inline netbeans.apache.org/src/content/about/oracle-transition.asciidoc:33]: Typo: infrastructure

Comment (vieiro) [reply netbeans.apache.org/src/content/about/oracle-transition.asciidoc]: Thanks, @KacerCZ , fixed in new commit.

**Comment by vieiro** [comment]
@jtulach Yep, this can be linked from elsewhere. We could redirect hg.netbeans.org to this page, or even the bits.netbeans.org/maven2, for instance. Let us know which ones you prefer.

**Comment by JaroslavTulach** [comment]
> We could redirect hg.netbeans.org to this page, or even the bits.netbeans.org/maven2, for instance. Let us know which ones you prefer.

I'd redirected both as this page now talk covers both - the Hg as well as Maven bits, right?

**Comment by vieiro** [comment]
@JaroslavTulach Yep. We're ready on our side ( https://github.com/apache/infrastructure-p6/pull/622/commits/25a4c7b627dd5edf2ade1776fa88b1cbb690c783 ), we'll meet with Apache Infra at Slack this EU afternoon at 16:30h CET to perform the DNS change and apply this pull request. After that we'll check the redirections. If everything goes well we'll close https://issues.apache.org/jira/browse/INFRA-21463 afterwards.

**Pull request merged**